packages/iov-bns/src/bnsconnection.ts

```diff
@@ -262,15 +262,18 @@ export class BnsConnection implements BcpAtomicSwapConnection {
     return dummyEnvelope(data);
   }
 
-  public async getAccount(account: BcpAccountQuery): Promise<BcpQueryEnvelope<BcpAccount>> {
+  public async getAccount(query: BcpAccountQuery): Promise<BcpQueryEnvelope<BcpAccount>> {
     let response: QueryResponse;
-    if (isAddressQuery(account)) {
-      response = await this.query("/wallets", decodeBnsAddress(account.address).data);
-    } else if (isPubkeyQuery(account)) {
-      const address = bnsCodec.keyToAddress(account.pubkey);
+    if (isAddressQuery(query)) {
+      response = await this.query("/wallets", decodeBnsAddress(query.address).data);
+    } else if (isPubkeyQuery(query)) {
+      const address = bnsCodec.identityToAddress({
```

Comment:
can you remove `bnsCodec.` as you do in `getNonce` below?

Reply:
Yes we can. `identityToAddress` and `bnsCodec.identityToAddress` are the exact same object